scripts/setup.bash

```diff
@@ -2,6 +2,8 @@
 export MIL_WS="$(realpath $(dirname $BASH_SOURCE)/../../..)"
 source /opt/ros/melodic/setup.bash
 source $MIL_WS/devel/setup.bash
+source $MIL_WS/src/mil/SubjuGator/scripts/bash_aliases.sh
+source $MIL_WS/src/mil/mil_common/ros_alarms/scripts/bash_aliases.sh
```

Comment:
NIce, been meaning to do this for a while

Reply:
Thanks